casepro/cases/views.py

```diff
@@ -413,15 +413,27 @@ def render_as_json(self, partners, with_activity):
                                                        *month_range(0)).scope_totals()
                 last_month = DailyCount.get_by_partner(partners, DailyCount.TYPE_REPLIES,
                                                        *month_range(-1)).scope_totals()
+                average_replied = DailySecondTotalCount.get_by_partner(partners,
+                                                                       DailySecondTotalCount.TYPE_TILL_REPLIED)
+                average_replied = average_replied.scope_averages()
```

Comment:
This should be within a given time window rather than an overall average, e.g. average response time in last month. I think it's more meaningful and it'll be more performant.